Source/WebInspectorUI/UserInterface/Views/CookieStorageContentView.js

```diff
@@ -611,6 +620,8 @@ WI.CookieStorageContentView = class CookieStorageContentView extends WI.ContentV
             return cookie.domain || missingValue;
         case "path":
             return cookie.path || missingValue;
+        case "partitionKey":
+            return cookie.partitionKey || missingValue;
```

Comment:
why do we need both `partitioned` and `partitionKey` if all we're showing in the UI is just the `partitionKey`?  is it not valid to assume that a cookie is `partitioned` if it's `partitionKey` is non-empty/non-null?  if so, should we have an additional column "Partitioned" that contains a checkmark if the cookie is `partitioned` regardless of it's `partitionKey`?

Reply:
CHIPS introduces a new Cookie attribute that is a boolean `Partitioned`, when that attribute is present we compute the appropriate `partitionKey` which is the site (schemeful registrable domain) of the top frame. I prefer avoiding computing the partition key in Web Inspector for the Cookie Popover, and just rely on the logic for computing it when we create the cookie. At that point, we can read the partition key from the cookie and show it in the Storage tab. This is why I added the Partitioned checkbox in the popover, and the partition key in the storage tab.